lib/ex_debug_toolbar/phoenix.ex

```diff
@@ -30,6 +30,8 @@ defmodule ExDebugToolbar.Phoenix do
         case dispatch_router(conn, opts) do
           %{private: %{phoenix_endpoint: ExDebugToolbar.Endpoint}} = conn ->
             conn
+          %{private: %{toolbar_ignore_path?: true}} = conn ->
```

Comment:
can you add a comment somewhere here, saying that this works by not starting the request, so all subsequent calls to add data are ignored?